client/src/main/kotlin/io/spine/examples/shareaware/client/Application.kt

```diff
@@ -75,7 +76,7 @@ public fun application(): Unit = application {
                 when (currentPage.value) {
                     Page.HOME -> Text("HOME")
```

Comment:
Let's make all cases lead to something called `...Page`. Not just render `Text(...)`.